src/controllers/order.controller.ts

```diff
@@ -1,16 +1,19 @@
 import { RequestHandler } from 'express';
+import { validate as uuidValidate } from 'uuid';
 import { ApiResponse } from '../types/shared';
 import { prismaClient } from '../helpers';
+import { OrderLog } from '@prisma/client';
 
 class OrderController {
   public static createOrderHandler: RequestHandler = async (req, res: ApiResponse, next) => {
     try {
-      const { type, status, orderMeals } = req.body;
+      const { type, status, orderMeals, reservationLogId } = req.body;
```

Comment:
use `yup` to validate input

reference
https://github.com/emily40830/point-pro-backend/blob/2e1ffdd2208cbe7f51a75736e7d5cc9387da1536/src/controllers/auth.controller.ts#L10

https://github.com/emily40830/point-pro-backend/blob/2e1ffdd2208cbe7f51a75736e7d5cc9387da1536/src/schemas/orders.schema.ts#L3

https://github.com/emily40830/point-pro-backend/blob/main/src/middlewares/validateOrderData.middleware.ts#L5